src/Dialogs/Device/FLARM/ConfigWidget.cpp

```diff
@@ -83,14 +44,14 @@ void
 FLARMConfigWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
                            [[maybe_unused]] const PixelRect &rc) noexcept
 {
-  RequestAllSettings(device);
+  PopupOperationEnvironment env; 
+  device.RequestAllSettings(flarm_setting_names, env);
 
   baud = GetUnsignedValue(device, "BAUD", 2);
-  priv = GetUnsignedValue(device, "PRIV", 0) == 1;
   thre = GetUnsignedValue(device, "THRE", 2);
-  range = GetUnsignedValue(device, "RANGE", 3000);
   acft = GetUnsignedValue(device, "ACFT", 0);
   log_int = GetUnsignedValue(device, "LOGINT", 2);
+  priv = GetUnsignedValue(device, "PRIV", 0) == 1;
```

Comment:
Why do you move the priv? its adding noise to the commit.

Reply:
I just wanted to hear your argument. :) Thank you.

Reply:
I will add my reasoning to the commit message, so it will be clear for a future reader.